lib/programmemory.cpp

```diff
@@ -534,6 +543,25 @@ void execute(const Token *expr,
         else
             *error = true;
     }
+    else if (Token::Match(expr->tokAt(-3), "%var% . %name% (")) {
+        const Token* containerTok = expr->tokAt(-3);
+        if (astIsContainer(containerTok) && containerTok->varId() > 0) {
```

Comment:
Redundant check `containerTok->varId() > 0`. The `%var%` only matches if varid is nonzero.